corehq/apps/app_manager/tests/test_bulk_app_translation.py

```diff
@@ -408,6 +408,66 @@ def test_empty_translations(self):
             ]
         )
 
+    def test_partial_case_list_translation_upload(self):
+        # note this isn't a "partial" upload because this app only has one case list property
+        module = self.app.get_module(0)
+        self.assertEqual(
+            module.case_details.short.columns[0].header, {'en': 'Name'}
+        )
+        translation_data = []
+        # filter out the case lists translation from the upload
+        for sheet in self.upload_no_change_data:
+            if sheet[0] != 'module1':
+                translation_data.append(sheet)
+                continue
```

Comment:
I think this would be a little easier to read as `if sheet[0] == 'module1':`, even though it's an extra level of indentation.